src/swagger-specs/data-hygiene.yaml

```diff
@@ -0,0 +1,1485 @@
+      responses:
+        200:
+          description: The dataset expiration was successfully canceled. 
+          content:
+            application/json:
+              example:
+                message: "The dataset was successfully canceled."
+        400:
+          x-summary: Bad Request
+          description: >
+            This can occur if the specified ID is invalid or missing required fields.
+          content:
+            application/json:
+              example:
+                error_code: "400 BAD REQUEST"
+                message: "The specified ID is invalid or missing required fields."
+        401:
+          x-summary: Unauthorized
+          description: >
+            This can occur if the authentication credentials are missing or invalid.
+          content:
+            application/json:
+              example:
+                error_code: "401 UNAUTHORIZED"
+                message: "Authentication credentials are missing or invalid."
+        403:
+          x-summary: Forbidden
+          description: >
+            This can occur if you do not have the necessary permissions to perform this action.
+          content:
+            application/json:
+              example:
+                error_code: "403 FORBIDDEN"
+                message: "User does not have the necessary permissions."
+        404:  
+          x-summary: Not Found
+          description: >
+            Occurs if the dataset expiration with the specified ID cannot be found.
+          content:
+            application/json:
+              example:
+                error_code: "404 NOT FOUND"
+                message: "The requested dataset expiration could not be found."
+        500:
+          x-summary: Internal server error
+          description: >
+            This can occur if there is an issue on the server while processing the request.
+          content:
+            application/json:
+              example:
+                error_code: "500 INTERNAL SERVER ERROR"
+                message: "An internal server error occurred while processing the request."
+
+  /workorder:
+    get:
+      tags:
+        - Record Delete
+      summary: List workorders
+      description: List all workorders.
+      operationId: getWorkOrders
+      parameters:
+        - $ref: '#/components/parameters/Authorization'
+        - $ref: '#/components/parameters/XApiKey'
+        - $ref: '#/components/parameters/XImsOrgId'
+        - $ref: '#/components/parameters/XSandboxName'
+        - $ref: '#/components/parameters/requestId'
+      responses:
+        200:
+          x-summary: "Success"
+          description: "Returns the details of the specified dataset expiration."
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/workorderResponse"
+
+    post:
+      tags:
+        - Record Delete
+      summary: Create a workorder
+      description: Create a new workorder.
```

Comment:
yeah, they call it the 'record delete' feature. But its the workorders endpoint. Its all about deleting identities. I think it would drive you crazy. It's also the Data Hygiene API for Advance Data Lifecycle Management :')